README.md

```diff
@@ -4,16 +4,21 @@ Salling Group owns Netto, Føtex, Bilka, Salling, and more.
 Through this SDK you will be able to find the name, opening hours, address, and more of stores.
 The requests are made through the Salling Group API which can be found [here](https://developer.sallinggroup.com/).
 
-You will need the module `sg-base-sdk` in order to authenticate.
+This SDK uses the [Traverser library](https://www.npmjs.com/package/@salling-group/pagination-traverser).
+
 You can get your credentials through the [developer portal](https://developer.sallinggroup.com/).
 
 ## Getting Started.
 The following example gets names of all Netto stores in the ZIP code 8200.
 You will need to get a JWT secret or Bearer token with access to the Stores API from the [developer portal](https://developer.sallinggroup.com/). 
 ```js
-const { SallingGroupAPI } = require('sg-base-sdk');
 const StoresSDK = require('sg-stores-sdk');
```

Comment:
In the Jobs SDK we just call this object `Jobs`. I don't feel strongly about either, but we should probably align the naming convention between the README files.